### PR TITLE
include location residency verification in ov process

### PR DESCRIPTION
**Pull request by harshaellanki**
IVL-182297467

### CODE REVIEW

File: config/client_config/dc/system/config/templates/features/aca_individual_market/eligibilities.yml [hunk 1]
```diff
@@ -51,6 +51,7 @@ registry:
               - :citizenship
               - :american_indian_status
               - :social_security_number
+              - :dc_residency
```
Comment (markgoho) [inline config/client_config/dc/system/config/templates/features/aca_individual_market/eligibilities.yml:54]: we need to find a different property name here--no keys or property names should reference a state specifically

can we use `state_residency` instead?

File: config/client_config/me/system/config/templates/features/aca_individual_market/eligibilities.yml [hunk 2]
```diff
@@ -126,6 +127,13 @@ registry:
             item: 'gid://enroll_app/Family::FamilyMember'
           - key: :evidence_ref
             item: 'gid://enroll_app/Person'
+      - key: :me_residency
```
Comment (markgoho) [inline config/client_config/me/system/config/templates/features/aca_individual_market/eligibilities.yml:130]: similar to above, we should not be using a named client/state in a key or property name

File: spec/domain/operations/eligibilities/build_determination_spec.rb [hunk 3]
```diff
@@ -154,6 +154,32 @@
...
+        effective_date: effective_date,
+        eligibility_items_requested: eligibility_items_requested
+      }
+    end
+
+    it 'should build evidences for residency' do
+      result = subject.call(required_params)
+      expect(result.success?).to be_truthy
+      result.value!.subjects.each do |subject|
+        expect(subject.last.dig(:eligibility_states, :aca_individual_market_eligibility, :evidence_states, :dc_residency)).to be_present
```
Comment (markgoho) [inline spec/domain/operations/eligibilities/build_determination_spec.rb:178]: specs must be client agnostic, we cannot reference specific states in specs

File: system/config/templates/features/aca_individual_market/eligibilities.yml [hunk 4]
```diff
@@ -126,6 +127,13 @@ registry:
             item: 'gid://enroll_app/Family::FamilyMember'
           - key: :evidence_ref
             item: 'gid://enroll_app/Person'
+      - key: :dc_residency
```
Comment (markgoho) [inline system/config/templates/features/aca_individual_market/eligibilities.yml:130]: should change this to `state_residency`

**Review by markgoho — DISMISSED** [review]

File: config/client_config/dc/system/config/templates/features/aca_individual_market/eligibilities.yml (hunk 1 above)
Comment (markgoho) [reply config/client_config/dc/system/config/templates/features/aca_individual_market/eligibilities.yml:54]: perhaps this shouldn't be something that we use the resource registry for?

another option would be to use an environment variable, e.g.

`STATE_RESIDENCY_VERIFICATION=dc_residency` or `STATE_RESIDENCY_VERIFICATION=me_residency`

and then wherever you need to use that value, reference it with `ENV['STATE_RESIDENCY_VERIFICATION']`

**Review by nisanthyaganti9 — APPROVED** [review]

**Pull request merged**